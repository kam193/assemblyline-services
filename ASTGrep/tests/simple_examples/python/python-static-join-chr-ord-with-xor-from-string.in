''.join(chr(ord(phi) ^ 3) for phi in "bT2ta0I3JKQoa[AnbT{oO@AygTIt`n:iY[MyO@Au`ztd`0oyJ@td`D")

''.join([chr(ord(phi) ^ 3) for phi in "bT2ta0I3JKQoa[AnbT{oO@AygTIt`n:iY[MyO@Au`ztd`0oyJ@td`D"])

''.join(([chr(ord(phi) ^ 3) for phi in "bT2ta0I3JKQoa[AnbT{oO@AygTIt`n:iY[MyO@Au`ztd`0oyJ@td`D"]))