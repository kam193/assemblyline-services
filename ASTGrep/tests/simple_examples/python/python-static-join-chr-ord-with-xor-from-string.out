'aW1wb3J0IHRlbXBmaWxlLCBzdWJwcm9jZXNzLCBvcywgc3lzICwgcG'

'aW1wb3J0IHRlbXBmaWxlLCBzdWJwcm9jZXNzLCBvcywgc3lzICwgcG'

'aW1wb3J0IHRlbXBmaWxlLCBzdWJwcm9jZXNzLCBvcywgc3lzICwgcG'